src/api/addon.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Methods related to the DocumentCloud note API
+ */
+
+import session from "./session";
+import { Addon } from "@/structure/addon";
+import { apiUrl } from "./base";
+import { queryBuilder } from "@/util/url";
+import { grabAllPages } from "@/util/paginate";
+import { DEFAULT_EXPAND } from "./common";
+
+export async function getAddons(id, expand = DEFAULT_EXPAND, page) {
+  // Returns addons for the specified document at the specified page
```

Comment:
It doesn't make sense both use `grabAllPages` as well as specify a page number.  We shouldn't have too many add-ons such that it matters at the moment, but we'll need to decide which one to use in which situations once we make more progress.